### DISCUSSION TITLE
Griglia dettaglio voti

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#142] [Richieste di aiuto] [answered]
Buongiorno,quando in qualità di docente cerco di stampare il foglio dettagliato dei voti ,intendo interrogazioni ,verifiche etc... , insomma, non esce un bel pdf in formato griglia,bensì il modello a tabella con alunni separati , si potrebbe integrare un nuovo modello a griglia normale ?Grazie

**Comment by trinko** [comment, marked as answer]
Forse in futuro, per momento non è una priorità.
